tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -578,12 +581,20 @@ def _convert(**args):
         init_logger('ERROR', False)
 
         argv.feManager = FrontEndManager()
-        ngraph_function = driver(argv)
+        ov_model, legacy_path = driver(argv)
+
+        # add MO meta data to model
+        ov_model.set_rt_info(get_version(), "MO_version")
+        ov_model.set_rt_info(get_rt_version(), "Runtime_version")
+        ov_model.set_rt_info(str(legacy_path), "legacy_path")
+        meta_info = get_meta_info(argv)
+        for key, value in meta_info.items():
+            ov_model.set_rt_info(str(value), ["conversion_parameters", str(key)])
```

Comment:
@slyalin also do we need all keys from meta info?

Reply:
Discussed it offline with @slyalin, we don't need keys that have default values or unset.